tink-docker/main.go

```diff
@@ -94,3 +96,33 @@ func DownloadFile(filepath string, url string) error {
 	_, err = io.Copy(out, resp.Body)
 	return err
 }
+
+func rebootWatch() {
+	fmt.Println("Starting Reboot Watcher")
+
+	// Forever loop
+	for {
+		if fileExists("/worker/reboot") {
+			//Build the command, and execute
+			cmd := exec.Command("/sbin/reboot")
```

Comment:
You could have some serious fun by doing things in `/proc`, but that probably isn't recommended.